scripts/citadel/launch/mint_post_kr.py

```diff
@@ -0,0 +1,171 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from decimal import Decimal
+from brownie import chain, Contract, interface
+
+# constants
+CITADEL_LAUNCH_DOLLAR_PRICE = 21
+SUPPLY_MULTIPLIER = Decimal(1666666666666666667 / 1e18)
+LIQUIDITY_PCT = 0.4
+TREASURY_PCT = 0.6
+MAX_DISCOUNT = 3000
+DISCOUNT = 1050
+RATES = [
+    593962000000000000000000,
+    591445000000000000000000,
+    585021000000000000000000,
+    574138000000000000000000,
+    558275000000000000000000,
+    536986000000000000000000,
+]
+# prices asset limits is going to be 1/3x and 3x
+FACTOR_PRICES = 3
+
+
+def mint_launch():
+    governance = GreatApeSafe(r.citadel.governance)
+    treasury = GreatApeSafe(r.citadel.treasury_vault)
+    governance.init_citadel()
+    governance.init_curve_v2()
+
+    # tokens involved
+    ctdl = interface.ICitadel(r.tokens.citadel, owner=governance.account)
+    xCTDL = governance.contract(r.tokens.xCTDL)
+    lp_ctdl_wbtc = governance.contract(r.tokens.crvCtdlWbtc)
+
+    # contracts involved
+    balance_checker = interface.IBalanceChecker(
+        r.helpers.balance_checker, owner=governance.account
```

Comment:
missing in current version of `addresses.py`

Reply:
added in last commits